MicrosoftFluentUI.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'MicrosoftFluentUI'
-  s.version          = '0.4.8'
+  s.version          = '0.5.102'
```

Comment:
where did 102 come from?

Reply:
I was using 0.5.100 as a test build number, and then I needed two more test builds :)

(This will also change to 0.5.0)

Reply:
Are you planning to change it back to 0.5.0 before the merge of this PR?